bindings/trycp_runner/src/macros.rs

```diff
@@ -1,13 +1,34 @@
 #[macro_export]
 macro_rules! embed_conductor_config {
     () => {
-        fn conductor_config() -> &'static str {
-            static CONDUCTOR_CONFIG: &str = include_str!("../../../conductor-config.yaml");
-            static CONDUCTOR_CONFIG_CI: &str = include_str!("../../../conductor-config-ci.yaml");
+        fn conductor_config() -> String {
+            static BASE_CONDUCTOR_CONFIG: &str =
+                include_str!("../../../conductor-config/conductor-config.yaml");
+            static BASE_CONDUCTOR_CONFIG_CI: &str =
+                include_str!("../../../conductor-config/conductor-config-ci.yaml");
+
+            static CHC_CONDUCTOR_CONFIG: &str =
+                include_str!("../../../conductor-config/with_chc.yaml");
 
             match std::env::var("CONDUCTOR_CONFIG") {
-                Ok(value) if value == "CI" => CONDUCTOR_CONFIG_CI,
-                _ => CONDUCTOR_CONFIG,
+                Ok(value) if value == "CI" => {
+                    if std::env::var("CHC_ENABLED")
```

Comment:
Would this be cleaner if it was a `let mut config = match` at the top level of this block and then the `if CHC_ENABLED` was a separate statement that might update the config before it is returned?